ApplicationLibCode/ProjectDataModel/RimContourMapProjection.cpp

```diff
@@ -75,8 +75,8 @@ RimContourMapProjection::RimContourMapProjection()
 {
     CAF_PDM_InitObject( "RimContourMapProjection", ":/2DMapProjection16x16.png" );
 
-    CAF_PDM_InitField( &m_relativeSampleSpacing, "SampleSpacing", 0.9, "Sample Spacing Factor" );
-    m_relativeSampleSpacing.uiCapability()->setUiEditorTypeName( caf::PdmUiDoubleSliderEditor::uiEditorTypeName() );
+    CAF_PDM_InitFieldNoDefault( &m_resolution, "Resolution", "Sampling Resolution" );
+    m_resolution.setValue( RimContourMapResolutionTools::SamplingResolution::NORMAL );
```

Comment:
Suggest default sampling resolution `Fine` to better match previous default value of 0.9.